src/cecs277TermProject/Logger.java

```diff
@@ -0,0 +1,28 @@
+package cecs277TermProject;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+/**
+ * Logger is simply our file I/O class, meant to write to the transaction log.
+ */
+public class Logger 
+{
+	
+	private File outFile;
+	private PrintWriter out;
+	
+	public Logger(File outFile) throws FileNotFoundException
```

Comment:
outFile should be initialized in here instead of let the caller side instantiate it. This constructor should take no argument.